app.js

```diff
@@ -1,38 +1,80 @@
+const inputLabel = document.getElementById('inputLabel');
 const userInput = document.getElementById('todoInput');
 const addBtn = document.querySelector('#addButton');
 const list = document.querySelector('#list');
 
-let listItem = document.createElement('li');
+let listItem = document.querySelectorAll('li');
+let doneBtn = document.querySelectorAll('#doneBtn');
+let deleteBtn = document.querySelectorAll('#deleteBtn');
 
 function addItem(event) {
 
-    // Create li
```

Comment:
Good code needs no comments, it should be easily reviewable and readable, if it is not, it is not good code! Good habbit to get into :) Only need comments if the algo is very complex or to warn a poor dev of some usecase no one considered.